### PR TITLE
Improve identity cache default

### PR DESCRIPTION
**Pull request by mffrench**
Signed-off-by: Mathilde Ffrench <mathilde.ffrench@fr.ibm.com>

### CODE REVIEW

File: platform/fabric/core/generic/config/config.go [hunk 1]
```diff
@@ -198,14 +198,16 @@ func (c *Config) GetPath(key string) string {
 	return c.configService.GetPath("fabric." + c.prefix + key)
 }
 
-func (c *Config) MSPCacheSize(defaultValue int) int {
+const DefaultCacheSize = 3
```
Comment (adecaro) [inline platform/fabric/core/generic/config/config.go:201]: Please, move this const to the top of the file.
Shall we call it DefaultMSPCacheSize?
Please, add also a comment describing the constant.

File: platform/fabric/core/generic/msp/cache.go [hunk 2]
```diff
@@ -35,7 +35,6 @@ func NewIdentityCache(backed IdentityCacheBackendFunc, size int) *IdentityCache
 		timeout: time.Millisecond * 100,
 	}
 	go ci.run()
-
```
Comment (adecaro) [inline platform/fabric/core/generic/msp/cache.go:38]: do we need to remove this file?

**Review by adecaro — CHANGES_REQUESTED** [review]

File: platform/fabric/core/generic/msp/cache.go (hunk 2 above)
Comment (mffrench) [reply platform/fabric/core/generic/msp/cache.go:38]: you mean this line I guess... don't think so ;)

File: platform/fabric/core/generic/config/config.go (hunk 1 above)
Comment (mffrench) [reply platform/fabric/core/generic/config/config.go:201]: OK ! making the change

**Review by adecaro — APPROVED** [review]

**Pull request merged**